### PR TITLE
DroneCI PR fix

### PR DESCRIPTION
**Pull request by TokRa13**
## Description:

**Related issue (if applicable):** fixes git behavior on DroneCI agent on PR

**Pull request in [tatau.io](https://github.com/TatauCloud/tatau/)

## Checklist:
  - [ ] The code change is tested and works locally.
  - [ ] Local tests pass with `tox`. **Your PR cannot be merged unless tests pass**
  - [ ] There is no commented out code in this PR.

If user exposed functionality or configuration variables are added/changed:
  - [ ] Documentation added/updated

If the code communicates with devices, web services, or third-party tools:
  - [ ] New or updated dependencies have been added to `setup.py.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -1,10 +1,17 @@
...
+    pull: true
+
 pipeline:
 
   typecheck35:
     pull: true
     group: py35
     image: tataucloud/python-cuda:3.5
     commands:
+      - git --no-pager log --oneline -n 3
```
Comment (27tech) [inline .drone.yml]: Can you explain this line? And why it's added only for typecheck35?

Comment (TokRa13) [reply .drone.yml]: this is to ensure that Drone got latest commit from PR, not the previous one.

Comment (TokRa13) [reply .drone.yml]: It's actually not required. This is just an additional info in droneCI stdout for us. Yes, first step is enough.

I believe we can safely remove this line once we are 100% sure everything works as expected.

**Comment by 27tech** [comment]
Resolve conflicts

**Pull request merged**